src/debugging.md

```diff
@@ -247,7 +252,7 @@ from being pressed to being released, it coincides with the
 object calling
 [`setState()`](https://docs.flutter.io/flutter/widgets/State/setState.html)
 and thus marking itself dirty. That is why if you look at the dump you
-will see that specific object marked "dirty". You can also see what
+see that specific object marked "dirty". You can also see what
```

Comment:
Since this is a conditional (has "if you"), shouldn't it be "would" instead of "will"?

As in "That is why, if you looked at the dump, you would see that..."

Or, if you don't want the conditional, it should be "This is why, when you look at the dump, you see that..."

Reply:
I suppose it's grammatically correct either way.  I think I like the last one best, without the "if".